app/src/main/java/no/usn/mob3000/ui/theme/Theme.kt

```diff
@@ -1,57 +1,46 @@
 package no.usn.mob3000.ui.theme
 
-import android.os.Build
 import androidx.compose.foundation.isSystemInDarkTheme
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.darkColorScheme
-import androidx.compose.material3.dynamicDarkColorScheme
-import androidx.compose.material3.dynamicLightColorScheme
 import androidx.compose.material3.lightColorScheme
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.platform.LocalContext
-
-private val DarkColorScheme = darkColorScheme(
-    primary = Purple80,
-    secondary = PurpleGrey80,
-    tertiary = Pink80
-)
-
-private val LightColorScheme = lightColorScheme(
-    primary = Purple40,
-    secondary = PurpleGrey40,
-    tertiary = Pink40
-
-    /* Other default colors to override
-    background = Color(0xFFFFFBFE),
-    surface = Color(0xFFFFFBFE),
-    onPrimary = Color.White,
-    onSecondary = Color.White,
-    onTertiary = Color.White,
-    onBackground = Color(0xFF1C1B1F),
-    onSurface = Color(0xFF1C1B1F),
-    */
-)
 
 @Composable
 fun ChessbuddyTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
-    // Dynamic color is available on Android 12+
-    dynamicColor: Boolean = true,
     content: @Composable () -> Unit
 ) {
-    val colorScheme = when {
-        dynamicColor && Build.VERSION.SDK_INT >= Build.VERSION_CODES.S -> {
-            val context = LocalContext.current
-            if (darkTheme) dynamicDarkColorScheme(context) else dynamicLightColorScheme(context)
-        }
+    val colors = if (darkTheme) {
+        darkColorScheme(
+            primary = DarkDefaultButton,
+            secondary = DarkNavbarButtonSelected,
+            background = DarkDefaultBackground,
+            surface = DarkNavbarBackground,
+            onPrimary = DarkProfileUserBackground,
+            onSecondary = DarkProfileUserStatisticsBackground,
+            onBackground = DarkProfileUserBackground,
+            onSurface = DarkProfileUserStatisticsBackground,
+        )
+    } else {
+        lightColorScheme(
+            primary = DefaultButton,
+            secondary = NavbarButtonSelected,
+            background = DefaultBackground,
+            surface = NavbarBackground,
+            onPrimary = DefaultBackground,
+            onSecondary = DefaultBackground,
+            onBackground = OnDefaultBackground,
+            onSurface = ProfileUserStatisticsBackground,
 
-        darkTheme -> DarkColorScheme
-        else -> LightColorScheme
+        )
```

Comment:
I'm pretty sure that this form of handling for color schemes are less dynamic than they were before? Regardless, I am of the opinion that the color schemes should be defined separately, and then call them in whichever logic that needs to set which theme is in use.